index.js

```diff
@@ -185,9 +193,53 @@ class Ora {
 			cliCursor.show(this.stream);
 		}
 
+		if (this.consumeStdin && process.stdin.isTTY) {
+			this.stopConsumingStdin();
+		}
+
 		return this;
 	}
 
+	startConsumingStdin() {
+		const me = this;
+		const {stdin} = process;
+
+		this._stdinOldRawMode = stdin.isRaw;
+		this._stdinOldEmit = stdin.emit;
+		this._stdinOldEmitOwnProperty = Object.prototype.hasOwnProperty.call(stdin, 'emit');
+
+		stdin.setRawMode(true);
+		stdin.on('data', noop);
+		stdin.emit = function (event, data) {
+			if (event === 'data') {
+				if (data.includes(0x03)) {
+					if (this.listenerCount('SIGINT') > 0) {
+						this.emit('SIGINT');
+					} else {
+						process.emit('SIGINT');
+					}
+				}
+			} else {
+				me._stdinOldEmit.apply(this, arguments); // eslint-disable-line prefer-rest-params
```

Comment:
```suggestion
				me._stdinOldEmit.call(this, event, data);
```

Reply:
I guess you misread the code there. It is applied in case it is NOT `data` event, so it may have arbitrary number of arguments.

Reply:
Ok. But my intention was to say that you should not use `arguments`. It's an anti-pattern. Use rest syntax instead.